src/ezdxf/addons/drawing/pymupdf.py

```diff
@@ -424,6 +429,46 @@ def draw_image(self, image_data: ImageData, properties: BackendProperties) -> No
             oc=self.get_optional_content_group(properties.layer),
         )
 
+    def register_font(self, font: AbstractFont) -> int:
```

Comment:
I guess this implementation cannot handle SHX fonts.

Reply:
I would assume so. I wonder what autocad / other cad programs do when exporting if non-ttf fonts are used?

If both exact rendering and selectable/interpretable text was desirable then invisible text could be layered on top of the baked glyphs hypothetically